cmd/build-disk.go

```diff
@@ -19,15 +19,15 @@ package cmd
 import (
 	"os/exec"
 
-	"github.com/rancher/elemental-toolkit/pkg/constants"
-	eleError "github.com/rancher/elemental-toolkit/pkg/error"
-	v1 "github.com/rancher/elemental-toolkit/pkg/types/v1"
+	"github.com/rancher/elemental-toolkit/v2/pkg/constants"
+	eleError "github.com/rancher/elemental-toolkit/v2/pkg/error"
+	"github.com/rancher/elemental-toolkit/v2/pkg/types"
```

Comment:
@frelon if I understood what you meant you were imagining something like that right? So no v1 or v2, only types which somehow also reflects we are not supporting both on the same code base.

Reply:
Yep, otherwise we could consider keeping the `pkg/types/v1` directory and only making changes in `v2`, but it feels like overkill to me :+1: